src/components/ImagePicker/index.scss

```diff
@@ -65,6 +65,17 @@ $modal-backdrop-opacity: 0.7;
         background-position: 50% 50%;
         z-index: 666;
     }
+
+    // Center the spinner both horizontally & vertically
+    .loading-spinner {
+        width: 50px;
+        height: 50px;
```

Comment:
Are `width` and `height` required here as the `size` prop is defined for the `CircularProgress` component?